BackEnd/src/main/java/com/exadel/carpoolfree/controller/MessageController.java

```diff
@@ -0,0 +1,46 @@
+package com.exadel.carpoolfree.controller;
+
+
+import com.exadel.carpoolfree.model.Message;
+import com.exadel.carpoolfree.repository.MessageRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/chat")
```

Comment:
I think we need to use `/api/messages` here have one name for every model object

Reply:
done